app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,32 @@
  *= require_tree .
  *= require_self
  */
+
+/*flash*/
+.alert-error {
+    background-color: #f2dede;
+    border-color: #eed3d7;
+    color: #b94a48;
+    text-align: left;
+}
+
+.alert-alert {
+    background-color: #f2dede;
+    border-color: #eed3d7;
+    color: #b94a48;
+    text-align: left;
+}
```

Comment:
Classes `.alert-error` and `.alert-alert` are duplicated.
Same duplicated `.alert-success` and `.alert-notice`.
2 ways: either change styles so they are different or delete duplicated styles.